trieste/bayesian_optimizer.py

```diff
@@ -667,14 +667,25 @@ def optimize(
                             "will be available."
                         ) from e
 
-                with Timer() as total_step_wallclock_timer:
```

Comment:
I have removed this bit as initial model fitting wallclock is now shown as step 0, so also excluded from total step time